sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -78,10 +78,10 @@
  * @param <InputT> the type of the {@link DoFn}'s (main) input elements
  * @param <OutputT> the type of the {@link DoFn}'s (main) output elements
  */
-public class DoFnTester<InputT, OutputT> {
+public class DoFnTester<InputT, OutputT> implements AutoCloseable {
   /**
    * Returns a {@code DoFnTester} supporting unit-testing of the given
-   * {@link DoFn}.
+   * {@link DoFn}. By default, uses {@link CloningBehavior#CLONE_PER_BUNDLE}.
```

Comment:
I used CLONE_PER_BUNDLE to more accurately represent the previous behavior of DoFnTester (to minimize breakages in people's pre-existing tests) but I guess this makes more sense, and I have a hard time imagining what kind of code one would write that would actually break.